todoist: add page (#6360)
tldr-pages/tldr: pages/common/todoist.md

# todoist

> > This program will let you use Todoist from the CLI.

Reviewer: ```suggestion
> Access Todoist from the command-line.
```

> More information: <https://github.com/sachaos/todoist>.

- Add a task:

> `todoist add "{{some_task}}"`

Reviewer: ```suggestion
`todoist add "{{task_name}}"`
```

> - Add a high priority task, with label, project, and due date:

Reviewer: ```suggestion
- Add a high priority task with a label, project, and due date:
```

> `todoist add "{{some_task}}" --priority {{1}} --label-ids "{{label_id}}" --project-name "{{project_name}}" --date "{{tmr 9am}}"`

Reviewer: ```suggestion
`todoist add "{{task_name}}" --priority {{1}} --label-ids "{{label_id}}" --project-name "{{project_name}}" --date "{{tmr 9am}}"`
```

for the date, can "{{tomorrow 9am}}" work?

Author: yes, that would work too, the todoist api uses natural language

> - Add the same task, in quick mode:

Reviewer: Duplicating?

Author: the `quick` command calls the todoist api directly, this method uses the todoist-cli commands and options

> `todoist quick '#{{project_name}} "{{tmr 9am}}" p{{1}} {{some_task}} @{{label_name}}'`

Reviewer: ```suggestion
`todoist quick '#{{project_name}} "{{tmr 9am}}" p{{1}} {{task_name}} @{{label_name}}'`
```

Should there be a space between the @

Author: no, the label has to be specified like "@labelname"

> - List all tasks, with header and pretty colors:

Reviewer: ```suggestion
- List all tasks with a header and color:
```

`todoist --header --color list`

- List all high priority tasks:

`todoist --header --color list --filter p{{1}}`

> - List today's tasks with high priority that have the label `{{label_name}}`:

Reviewer: ```suggestion
- List today's tasks with high priority that have the a specified label:
```
